examples/test-c-client.c

```diff
@@ -0,0 +1,88 @@
+/*
+ * SRT - Secure, Reliable, Transport
+ * Copyright (c) 2017 Haivision Systems Inc.
+ * 
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * 
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; If not, see <http://www.gnu.org/licenses/>
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <arpa/inet.h>
+#include <unistd.h>
+
+#include "srt.h"
+
+int main( int argc, char** argv )
+{
+    int ss, st;
+    struct sockaddr_in sa;
+    int yes = 1;
+    const char message [] = "This message should be sent to the other side";
+
+    printf("srt startup\n");
+    srt_startup();
+
+    printf("srt socket\n");
+    ss = srt_socket(AF_INET, SOCK_DGRAM, 0);
+    if ( ss == SRT_ERROR )
+    {
+        fprintf(stderr, "srt_socket: %s\n", srt_getlasterror_str());
+        return 1;
+    }
+
+    printf("srt remote address\n");
+    sa.sin_port = htons(atoi(argv[2]));
+    if ( inet_pton(AF_INET, argv[1], &sa.sin_addr) != 1)
+    {
+        return 1;
+    }
+
+    printf("srt setsockflag\n");
+    srt_setsockflag(ss, SRTO_SENDER, &yes, sizeof yes);
```

Comment:
Mind how the user would like to call this application, and whether the stream would be sent from client to server or the other way around. Might be that it would be specified by options, and if not (for simplification) it's rather better that we have a kinda "stream server", so server is sending, client is reading. Important to inform the user that this flag is only needed to communicate with SRT with version earlier than 1.3.0.

Reply:
The user doesn't need to call this application. It does "dummy" work. This it's only an "example" code.

Reply:
Yes, but you know that users will next try to compile it and see if this works...

Reply:
Me too! :smile: